src/content/en/tools/workbox/guides/_shared/generate-sw-string-schema.html

```diff
@@ -44,7 +46,9 @@
     <p>
       This only needs to be set if you want to precache local files matching <code>glob</code>
       patterns. It does not need to be set if you have <code>self.__precacheManifest</code> defined
-      via a file included in <code>importScripts</code>.
+      via a file included in <code>importScripts</code>. In other words, developers using the
```

Comment:
Are all of these going to be added to webpack plugin *and* workbox build and CLI? If so, could we find some way to moving this info into a condition template that is only displayed for the webpack plugin?

Also feeling uneasy that we need a "In other words" rather than just stating what is easiest to understand.

Reply:
We don't expose `generateSWString` via the CLI.

We do expose it via `workbox-build` and yes, it would show up in those docs.

I can change the conditional language by creating a standalone `_shared/webpack-options.html` instead of including it in this file.